app/js/background.js

```diff
@@ -1,11 +1,11 @@
 // Copyright 2015 The Beagle term Authors. All rights reserved.
 // Use of this source code is governed by MIT LICENSE.
 
-chrome.app.runtime.onLaunched.addListener(function() {
+chrome.app.runtime.onLaunched.addListener(function () {
   new BeagleWindow();
 });
 
-var BeagleWindow = function() {
+var BeagleWindow = function () {
```

Comment:
that's fine. My code editor apparently uses a different style guide. I've gotta figure out how to change that. Anyways, give me a few mins and I'll commit a fix.